src/test/java/com/group12/LaunchInterceptorConditionCollectionTest.java

```diff
@@ -267,4 +267,75 @@ public void givenLengthIsLessThan0_whenLIC7_thenIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class,
                 () -> launchInterceptorConditionCollection.LIC7(points, kPts, length1));
     }
+
+    // Tests for LIC #10
+
+    @Test
+    public void givenThreePointsWithAreaGreaterThan1AndAreaIs1AndEPtsEqFPtsEq1_whenLIC10_thenTrue() {
```

Comment:
Just a small detail but it should say `givenFivePoints...` 🙂

Reply:
That's on purpose, because it's three points forming an area greater than 1

Reply:
I see, you are correct.